specreduce/core.py

```diff
@@ -2,3 +2,246 @@
 # This module implements the base CCDPROC functions
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+import os
+# import re
+# from copy import deepcopy
+# from datetime import datetime
+# from importlib import import_module
+# import warnings
+
+import matplotlib.pyplot as plt
+# import matplotlib.cm as cm
+# from matplotlib.widgets import Cursor
+
+# from astropy.convolution import convolve, Box1DKernel
+# from astropy import units as u
+# from astropy.io import fits
+# from astropy.modeling import models, fitting
+# from astropy.stats import sigma_clip
+# from astropy.table import Table
+# from astropy.wcs import WCS
+
+import numpy as np
+
+# from scipy.optimize import curve_fit
+# import scipy.signal
+from scipy.interpolate import UnivariateSpline
+# from scipy.interpolate import BSpline, SmoothBivariateSpline
+
+# from specutils import SpectralRegion
```

Comment:
Once complete, the imports can be cleaned up.

Reply:
Sure, will keep that in mind